packages/aws-rfdk/lib/deadline/scripts/python/configure_identity_registration_settings.py

```diff
@@ -223,29 +220,79 @@ def validate_config(config):
 ####################################
 
 
-def fetch_secret(secret, binary=False):
+def aws_cli(args: List[str]):
+    """
+    Run an AWS CLI command and return JSON parsed output
+    """
+
+    try:
+        text_output = subprocess.check_output(['aws'] + args, text=True)
+    except subprocess.CalledProcessError as e:
+        raise Exception(f"failed to call AWS CLI ({e.returncode}): \n{e.stdout}\n\n{e.stderr}")
+
+    json_obj = json.loads(text_output)
```

Comment:
Maybe a bit nitpicky... should this be in a try-except as well in case there's a parsing error?

Reply:
Done. I've also changed the code to [chain the original exceptions](https://docs.python.org/3/library/exceptions.html#exception-context) to the user-friendly ones.